apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/PartitionOptions.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using System.Threading;
+
+namespace Google.Cloud.Spanner.Data;
+
+/// <summary>
+/// An immutable class representing partition options.
+/// </summary>
+/// <remarks>
+/// <para>The specified values of `PartitionSizeBytes` and `MaxPartitions`
+/// are utilized in the RPC call while generating the partition tokens.</para>
+/// <para>The specified flag for `DataBoostEnabled` will be stored in the request property
+/// of the <see cref="CommandPartition"/> object and is used when executing the
+/// partition object.For more information see
```

Comment:
If you *do* want to refer to the properties in the protos, you can do so with a `<see cref="..." />` - but I probably wouldn't do that anyway. If there's product documentation, refer to that - if there isn't, just remove the sentence.

(If we keep the sentence, add a space after the period.)

Personally I find this sort of documentation more readable if you put the `<para>` opening and closing tags on their own lines - that way the fact that it's in two paragraphs becomes more obvious in the source code.